addons/danger/functions/fnc_tacticsAssess.sqf

```diff
@@ -118,7 +119,7 @@ if !(_enemies isEqualTo [] || {_unitCount < random 3}) then {
         !_speedMode
         && {_unit distance2D _x > RANGE_LONG}
         && {_unit knowsAbout _x < 2}
-        && {((getPosASL _x) select 2) > ((_eyePos select 2) + 15)}
+        && {((getPosWorld _x) select 2) > ((_eyePos select 2) + 15)}
```

Comment:
Eyepos reports ASL back. This will become an issue over water.

Reply:
Perhaps exit all tactics over or in water?

Reply:
when a object is place over water that position may also be water is the issue(for example the piers are sometimes overwater)